editor/src/core/workers/parser-printer/uid-fix.ts

```diff
@@ -143,10 +144,7 @@ function zipTopLevelElements(
   firstComponents.forEach((firstComponent, index) => {
     if (secondComponents.length > index) {
       const secondComponent = secondComponents[index]
-      const firstUID = getUtopiaID(firstComponent.rootElement)
-      const secondUID = getUtopiaID(secondComponent.rootElement)
 
-      onElement(firstUID, secondUID, EP.emptyElementPathPart, EP.emptyElementPathPart)
```

Comment:
Is this correct? We're never calling `onElement` on top level elements now?

Reply:
yes, this was using the root element, as the first call of `walkElementChildren` in the next line too, inside the function it's calling `onElement` on the root element and for fragments it skips to its children